package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@govconnex/ui",
-  "version": "0.0.181",
+  "version": "0.0.187",
```

Comment:
Since this number is higher than what is published in #142 , we need to merge that PR first and include all the changes from that PR

Reply:
i believe it is okay since i pulled from main

Reply:
We need to merge PR #142 first

Reply:
Okay that PR has been merged, do you want to rebase on main again, then should be good to go?